### ISSUE TITLE
VERBOSE - a new log level between INFO and DEBUG

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#1055]
Add a new log level between `LOG_INFO` and `LOG_DEBUG` - `LOG_VERBOSE`.
Try to don't break compatibility in box.cfg { log_level= xx }.

Depends on #1054.

**Comment by kostja** [comment]
Duplicate of gh-2467
